pacemaker.spec.in

```diff
@@ -630,6 +653,7 @@ exit 0
 
 %config(noreplace) %{_sysconfdir}/sysconfig/pacemaker
 %if %{defined _unitdir}
+%ghost %{_localstatedir}/lib/rpm-state/%{name}
```

Comment:
btw. tried multiple owning of the %ghosted directory and it is working as expected:
after removing the last packet owning the %ghosted directory it is properly removed